code/modules/jobs/job_types/station/medical/psychiatrist.dm

```diff
@@ -24,9 +24,7 @@
 					ails the mentally unwell, frequently under Security supervision. They understand the effects of various psychoactive drugs."
 	alt_titles = list(
 		"Psychologist" = /datum/prototype/struct/alt_title/psychologist,
-		"Psychoanalyst" = /datum/prototype/struct/alt_title/psychologist/psychoanalyst,
-		"Counselor" = /datum/prototype/struct/alt_title/counselor,
-		"Therapist" = /datum/prototype/struct/alt_title/therapist
```

Comment:
curious
why are we removing therapist but adding counselor

latter sounds even less like a proper medical role